tests_and_analysis/test/euphonic_test/test_sampling.py

```diff
@@ -1,16 +1,19 @@
 from collections.abc import Iterator
-from functools import partial
 
 import numpy as np
 import pytest
 
 import euphonic.sampling
 
+# @pytest.fixture
+# def fix_np_rng(monkeypatch):
+#         monkeypatch.setattr(np.random, 'default_rng',
+#                             partial(np.random.RandomState, seed=0))
+
+def rng_seed_0():
```

Comment:
Does this return a new generator or does it just affect the global state? I think the former from what I read, but best to just make sure.

Also, this approach seems to be considered [legacy](https://numpy.org/doc/stable/reference/random/legacy.html#numpy.random.RandomState)